mongodb/mongo_setup.py

```diff
@@ -0,0 +1,30 @@
+from mongoengine import connect, disconnect
+from pydantic_settings import BaseSettings
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
If you are using BaseSettings, why do you need this?. Also, **NEVER CALL FUNCTIONS OUTSIDE OF A FUNCTION!!!!!!**